### ISSUE TITLE
When scroll mouse-wheel make font more bigger, which cause line become very wide.

### ISSUE THREAD

**Issue by zw963** [flameshot-org/flameshot#699]
**Flameshot version**

```sh
 ╰─ $ flameshot --version
Flameshot v0.6.0
Compiled with Qt 5.12.1
```

**Describe the bug**

When i increasing font size, which make line tools became very wide.

![image](https://user-images.githubusercontent.com/549126/76876446-e7275680-68ac-11ea-9767-4e3312cac467.png)

**Expected behavior**
What i expected is:

![image](https://user-images.githubusercontent.com/549126/76876722-4edda180-68ad-11ea-865a-94783ba36bd2.png)

**System Information**
Arch Linux

Thank you.

**Comment by borgmanJeremy** [comment]
You need to use the scroll wheel in the other direction to decrease the size.

**Comment by zw963** [comment]
> You need to use the scroll wheel in the other direction to decrease the size.


@borgmanJeremy , i think i knew it. the issue is, why the wheel control the width of all components?
e.g. i want to a more bigger font,  but with more narrow line, for this case, it give a bad experience.

consider try https://github.com/linuxdeepin/deepin-screenshot, it behavior as what i want.

Thank you.

**Comment by zw963** [comment]
@borgmanJeremy , sorry, if i describe my issue in previous comment not so clear, right?

thank you.

**Comment by borgmanJeremy** [comment]
@zw963 Yes I understand the issue. This will be fixed when #782  is complete.
